packages/web-components/fast-foundation/src/breadcrumb/breadcrumb.template.ts

```diff
@@ -0,0 +1,20 @@
+import { html, slotted, elements } from "@microsoft/fast-element";
+import { Breadcrumb } from "./breadcrumb";
+
+/**
+ * The template for the {@link @microsoft/fast-foundation#Breadcrumb} component.
+ * @public
+ */
+export const BreadcrumbTemplate = html<Breadcrumb>`
+    <template
+        role="navigation"
+        aria-label="${x => x.ariaLabel}"
+        aria-labelledby="${x => x.ariaLabelledby}"
+    >
+        <div role="list">
```

Comment:
This is missing a part and class. If you place a class on this and set the value to `display: flex` that will fix your problem when one (but not all) items has start or end content.

Reply:
thank you!